Sources/Testing/Traits/Tag.Color+Loading.swift

```diff
@@ -8,10 +8,10 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
-private import TestingInternals
+@_implementationOnly import TestingInternals
 
 #if canImport(Foundation)
-private import Foundation
+import Foundation
```

Comment:
This can have `@_implementationOnly`, right?

Reply:
Specifically:

```
/Volumes/Dev/Source/swift-testing/Sources/Testing/Traits/Tag.Color+Loading.swift:14:22: warning: using '@_implementationOnly' without enabling library evolution for 'Testing' may lead to instability during execution
@_implementationOnly import Foundation
```

Reply:
Added a TODO comment.